### PR TITLE
Display the correct button based on the user platform

### PR DESCRIPTION
**Pull request by scandar**
Displays `⌘` on Mac and `CTRL` for other platforms in the command palette.

## Preview
Windows
image

Mac
image

### CODE REVIEW

File: src/hooks/useOS.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from "react";
+
+export type OS = "mac" | "other";
+
+export const useOS = () => {
+  const [os, setOs] = useState<OS>("mac");
```
Comment (BartoszJarocki) [inline src/hooks/useOS.ts:6]: Handling this doesn't require a state, actually! 

here's the revised code
```
export type OS = 'mac' | 'windows' | 'linux' | 'other';

export const useOS = (): OS => {
  const getOperatingSystem = (): OS => {
    const platform = window.navigator.platform.toLowerCase();

    if (platform.includes('mac')) return 'mac';
    if (platform.includes('win')) return 'windows';
    if (platform.includes('linux')) return 'linux';
    
    return 'other';
  };

  return useMemo(() => getOperatingSystem(), []);
};
```

other platforms are not needed, so it's ok to just use `mac` and `other` in this case, but the `useMemo` is an important part. It'll run only once and won't cause the rerender of component using this hook, as it's the case in an option with using state.

**Review by BartoszJarocki — CHANGES_REQUESTED** [review]

File: src/hooks/useOS.ts (hunk 1 above)
Comment (scandar) [reply src/hooks/useOS.ts:6]: `window.navigator.platform` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/platform), I would stay away of using it.

**Comment by VladimirCreator** [comment]
@scandar I think it also requires changing the "⌘" icon to "CTRL" text when screen resizes to mobile size. Will you implement it?

**Comment by scandar** [comment]
> @scandar I think it also requires changing the "⌘" icon to "CTRL" text when screen resizes to mobile size. Will you implement it?

I don't think that's the same thing, the `⌘` icon on mobile is a clickable button that could be anything it doesn't necessarily have to match the platform. IMO i think displaying the word `CTRL` in this circular button is going to look u
Screenshot 2024-01-16 at 3 33 09 PM
gly UI/UX wise.

**Review by BartoszJarocki — APPROVED** [review]
looks good, thank you!

**Pull request merged**